src/server/scripts/Outland/GruulsLair/boss_gruul.cpp

```diff
@@ -325,4 +312,4 @@ void AddSC_boss_gruul()
     RegisterSpellScript(spell_gruul_ground_slam_trigger);
     RegisterSpellScript(spell_gruul_shatter);
     RegisterSpellScript(spell_gruul_shatter_effect);
-}
+}
```

Comment:
I think it needs a blank line at the end of the file, seems to be complaining about it